### PR TITLE
Copying document sets collection based off new uri

### PR DESCRIPTION
**Pull request by anthonyhashemi**
## Changes in this PR:
- Renamed `copy_judgment` to `copy_document`
- `copy_document` now adds the document to the appropriate collection based on the uri.
- updated changelog

## Trello card / Rollbar error (etc)
https://trello.com/c/0Q5fI6gA/1072-ingesting-uri-parse-failures-not-being-added-to-judgment-or-press-summary-collection

### CODE REVIEW

**Comment by dragon-dxw** [comment]
This will be a breaking change for the editor -- recommend we have an alias for the original name which we can remove later.

**Comment by anthonyhashemi** [comment]
> This will be a breaking change for the editor -- recommend we have an alias for the original name which we can remove later.

discussed on slack - I will update editor ui once this is merged in.

**Review by jacksonj04 — APPROVED** [review]
Happy this looks good, just need to make sure EUI fixes happen in step with the version bump.

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ The format is based on [Keep a Changelog 1.0.0].
 
 ## [Unreleased]
 
+- Renamed `copy_judgment` to `copy_document`
```
Comment (jacksonj04) [inline CHANGELOG.md:9]:
```suggestion
- **Breaking:** Renamed `copy_judgment` to `copy_document`
```

Comment (anthonyhashemi) [reply CHANGELOG.md]: added

**Pull request merged**